src/wp-includes/script-loader.php

```diff
@@ -1180,6 +1180,9 @@ function wp_default_scripts( $scripts ) {
 		)
 	);
 
+	$scripts->add( 'password-toggle', "/wp-admin/js/password-toggle$suffix.js", array(), false, 1 );
+	$scripts->set_translations( 'password-toggle' );
+
```

Comment:
Minor nitpick, could this block be moved after `$scripts->set_translations( 'password-strength-meter' )`, to line 1165?

Reply:
Will do. Also noticed that `wp-i18n` is not listed as a dependency, even though I referenced it in `password-toggle.js`. Am I correct to think that this needs to be added as a dep?

Reply:
> Also noticed that `wp-i18n` is not listed as a dependency, even though I referenced it in `password-toggle.js`.

The `set_translations()` method ensures it's included in the dependencies so you don't need to include it here.

https://github.com/WordPress/wordpress-develop/blob/6036ee03af6aba166ab86b0dd9357dff2e2128da/src/wp-includes/class-wp-scripts.php#L576-L589